core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -688,6 +691,8 @@ angular.module('oppia').directive('conversationSkin', [
           };
           var _addNewCard = function(newCard) {
             PlayerTranscriptService.addNewCard(newCard);
+            CommandExecutorService.sendInteractionToOuterFrame(
```

Comment:
CommandExecutorService.onLearnerViewCardTransition(...)

Can you do this without private methods? Doesn't the card object have a getInteractionId() defined on it?